modules/ROOT/pages/environments-concept.adoc

```diff
@@ -20,7 +20,7 @@ When you navigate to API Manager, you see the environment control bearing the na
 
 image::environment.png[]
 
-When you click the environment control, a list of your environments appears.
+When you click the environment control, a list of your production and sandbox environments appears.
```

Comment:
```suggestion
When you click the environment control, a list of your production and sandbox environments appear.
```

Reply:
@priyaayyar 
Isn’t the subject, “a list”, singular, therefore making the verb also singular (“appears”)? I’m thinking the verb isn’t “appear”, as this doesn’t agree with the subject, “a list”.